.changeset/hot-pumpkins-clean.md

```diff
@@ -1,26 +1,40 @@
 ---
```

Comment:
Updated the changeset note we wrote when introducing `Shape`, downgraded version bump from "minor" to "patch" since we are not breaking compatibility anymore.